Missing export of INTL_RELATIVE_TIME_PIPE_DEFAULT_OPTIONS in "angular-ecmascript-intl"

When trying to inject  INTL_RELATIVE_TIME_PIPE_DEFAULT_OPTIONS token in App Module providers.

Output: 
Error: Module not found: Error: Package path ./lib/relative-time/relative-time-pipe-default-options is not exported from package /home/addiel/Documents/Proyectos/plaza_frontend/node_modules/angular-ecmascript-intl (see exports field in /home/addiel/Documents/Proyectos/plaza_frontend/node_modules/angular-ecmascript-intl/package.json)

![Screenshot from 2024-04-09 15-15-14](https://github.com/json-derulo/angular-ecmascript-intl/assets/105292535/846b95a0-abdc-4582-9dca-9f2859b14113)

export should be below export * from './lib/relative-time/relative-time.pipe';

My Angular's version is 17.3.3 and Angular Intl latest 3.0.0

--- a/projects/angular-ecmascript-intl/src/public-api.ts
+++ b/projects/angular-ecmascript-intl/src/public-api.ts
@@ -18,6 +18,7 @@ export * from './lib/list/intl-list.pipe';
 export * from './lib/locale';
 export * from './lib/percent/intl-percent-pipe-default-options';
 export * from './lib/percent/intl-percent.pipe';
+export * from './lib/relative-time/relative-time-pipe-default-options';
 export * from './lib/relative-time/relative-time.pipe';
 export * from './lib/unit/intl-unit-pipe-default-options';
 export * from './lib/unit/intl-unit.pipe';

